src/api/app/views/webui/project/edit.js.haml

```diff
@@ -1 +1,7 @@
-$('.in-place-edition').html("#{escape_javascript(render(partial: 'webui/project/edit', locals: { project: @project }))}");
+:plain
```

Comment:
You would not have to do this if you just skip the `.haml` pre-processor.

Reply:
Ok, `*.js.haml` has been converted to `*.js.erb` and the `:plain`has been removed for both edit and show.